views/deleteForm.html

```diff
@@ -0,0 +1,15 @@
+<html>
```

Comment:
as a next step, you could consider using something like http://swagger.io/ as the api interface, I find stuff like that really helps you to think about your api from a user perspective